src/shared_state/conflicts.md

```diff
@@ -0,0 +1,52 @@
+# Detecting conflicts
+
+Modifications to shared state can be considered conflicting when two conditions are met:
+
+1. They affect an overlapping section of a data structure, such as if two users both try to set the title of the same todo-list item
+
+2. They are [concurrent](https://en.wikipedia.org/wiki/Concurrency_(computer_science)) - meaning that the participants in the system cannot conclusively determine the order in which they occurred
+
+_Concurrent updates which affect unrelated data_ can be resolved without issue, as can _sequential updates to related data_.
+
+Concurrency can effectively be ignored from a client's perspective in schemes that use a central server as a single source of truth.
+In practice one message will always arrive slightly before another, and even if they were truly simultaneous the server could arrange them into an arbitrary order.
+Because this can be done for all messages, all updates can be said to have a [total order](https://en.wikipedia.org/wiki/Total_order), which makes it possible for all clients to compute the same results when they are applied sequentially.
+
+## Conflict resolution in webxdc applications
+
+Many webxdc applications are incapable of producing conflicts, either because they do not use the network communication APIs, or because any given data structure can only have one writer.
+It is worth considering that the webxdc host application might permit a single user to use multiple devices,
+though it is ultimately up to an app's developers whether they want to support such use-cases.
+
+For applications that permit multiple devices to update the same values using [`webxdc.sendUpdate()`](../spec/sendUpdate.html#sendupdate) it is necessary to provide some mechanism through which all clients can determine a total order for all updates.
```

Comment:
```suggestion
Webxdc applications can use [`webxdc.sendUpdate()`](../spec/sendUpdate.html#sendupdate) and [`setUpdateListener`](https://docs.webxdc.org/spec/setUpdateListener.html) to send and receive data to and from other devices. It then is necessary to provide some mechanism through which all clients can determine a total order for all updates, or can otherwise arrive at a shared consistent view on different devices, once all messages have been delivered.   
```